packages/devextreme/testing/tests/DevExpress.ui.widgets/chatParts/confirmationPopup.markup.tests.js

```diff
@@ -0,0 +1,34 @@
+import $ from 'jquery';
+
+import ConfirmationPopup, {
+    CHAT_CONFIRMATION_POPUP_WRAPPER_CLASS,
+} from '__internal/ui/chat/confirmationpopup';
+
+const DX_POPUP_CONTENT_CLASS = 'dx-popup-content';
+
+const DX_BUTTON_CLASSNAME = 'dx-button';
+
+const moduleConfig = {
+    beforeEach: function() {
+        this.$element = $('#component');
+    },
+};
+
+QUnit.module('ConfirmationPopup', moduleConfig, () => {
+    QUnit.module('Render', moduleConfig, () => {
+        QUnit.test('Should render confirmation popup content and toolbar with correct items', function(assert) {
+            assert.expect(2);
+
+            const confirmationPopup = new ConfirmationPopup(this.$element, {}, { container: this.$element });
```

Comment:
Lets extract this part to 'reinit(), init()' methods like in other test files done